README.md

```diff
@@ -426,17 +428,27 @@ You can also choose the `services.cloud.sap.com/forceRotate` annotation (value d
 
 [Back to top](#sap-business-technology-platform-sap-btp-service-operator-for-kubernetes)
 
-## Multitenancy
-You can configure the SAP BTP service operator to work with more than one subaccount in the same Kubernetes cluster. This means that different namespaces can be connected to different subaccounts.
-The association between a namespace and a subaccount is based on a different set of credentials configured for different namespaces.
+## Working with Multiple Subaccounts
+
+By default, a Kubernetes cluster is related to one subaccount (see step 4 of the [Setup](#setup) section.
+The SAP BTP service operator can also be configured to manage multiple subaccounts within a single Kubernetes cluster. In this more complex use case, each resource is linked to a specific subaccount using a unique set of credentials. To connect a resource to a subaccount:
```

Comment:
the operator does not 'manage' subaccounts, but 'work with'. as described in the header of this section.

Reply:
done